Aikido.Zen.DotNetCore/Zen.cs

```diff
@@ -40,5 +40,20 @@ public static User GetUser()
             var contextAccessor = _serviceProvider.GetService(typeof(ContextAccessor)) as ContextAccessor;
             return contextAccessor?.CurrentUser;
         }
+
+        /// <summary>
+        /// Gets the current status of the Aikido Zen agent, including heartbeat reporting status.
+        /// This method provides a snapshot of the agent's communication status with the Zen API.
+        /// </summary>
+        /// <returns>
+        /// An <see cref="AgentStatus"/> object containing the current status information, including:
+        /// - Heartbeat reporting status indicating whether the agent is successfully communicating with the Zen API
+        /// - Success/failure state of recent API communications
+        /// - Whether heartbeat reports have expired or are current
+        /// </returns>
+        public static AgentStatus Status()
+        {
+            return Agent.Instance?.GetCurrentStatus();
+        }
```

Comment:
Shall we add some docs in `/docs` like https://github.com/AikidoSec/firewall-node/blob/main/docs/esbuild.md? What you wrote in PR description basically?

Reply:
Wouldn't document this too much tbh. Since it only checks for the heartbeat being successful and doesn't indicate if Zen is actually working fine or not.